### ISSUE TITLE
Renaming

### ISSUE THREAD

**Issue by skules** [riderkick/FMD#43]
So i use an app called mangablitz to read/download manga on my phone. Not all of the manga's have a download all option so i want to bypass that by using FMD and then just transferring it onto my phone.

The issue comes from the file name. For mangablitz to go from chapter to chapter the files must be named "Chapter 001" + "Chapter 002" and so forth.

Currently on FMD the %numbering% uses 0001, an extra 0 screws it up.

TL:DR - %numbering% from 0001 to 001 or something like that.

**Comment by riderkick** [comment]
How come sorting fixed digit like that can be messed up. There is should not be any problem even if there is thousand zero padding. It was the fault of app you're using.
Most of the series come with their own chapter number. You can just use "%CHAPTER%" and there is volume and chapter digit padding for chapter name. You can adjust it with your need.
![sshot-2](https://cloud.githubusercontent.com/assets/4972676/8015886/5ef4ede4-0c0f-11e5-96ff-b034d0c79145.jpg)
You may consider to use another manga reader. Currently I'm using Perfect Viewer and Comitton on my Android tablet/phone.
